spec/policy_machine_storage_adapters/active_record_spec.rb

```diff
@@ -451,6 +451,45 @@
       end
     end
 
+    describe '#pluck_from_descendants' do
+      context 'no filter is applied' do
+        it 'returns appropriate descendants and the specified attribute' do
+          plucked_results = [{ color: 'green' }, { color: 'green' }, { color: 'green' }]
+          expect(user_1.pluck_from_descendants(fields: [:color])).to match_array(plucked_results)
+        end
+
+        it 'returns appropriate descendants and multiple specified attributes' do
+          plucked_results = [
+            { unique_identifier: 'user_attr_1', color: 'green' },
```

Comment:
Mind making at least one of these not `green` so that we can see it's actually associating the correct attributes and values?

Reply:
Same comment below.